### DISCUSSION TITLE
why are we limited to making 100 cobbler sync changes a day?

### DISCUSSION THREAD

**Discussion by phrend** [cobbler/cobbler#2589] [General]
I see that the last 2 digits of the zone file serial number can't increment past 99 in a day, and was wondering if anyone knows why this limit was chosen? I'd like to increase the number, but don't know if this logic is here for a technical reason, or if the thought was just that nobody would need to make more than 100 changes in a day, so two digits were allocated, and this test just makes sure that the code doesn't overrun the 2 digits? (in other words - can i allocate 4 digits, and update the code accordingly without running into any problems?)

https://github.com/cobbler/cobbler/blob/master/cobbler/modules/managers/bind.py#L523

**Comment by SchoolGuy** [comment]
Great that you asked this. The short answer is "historically grown". Open a Feature request for it and we fix it. :)

**Reply by phrend** [reply]
Thank you! :)
https://github.com/cobbler/cobbler/issues/2592
